x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx

```diff
@@ -314,37 +300,22 @@ const AlertSummaryViewComponent: React.FC<{
   title?: string;
 }> = ({ browserFields, data, eventId, isDraggable, timelineId, title }) => {
   const summaryRows = useMemo(
-    () => getSummaryRows({ browserFields, data, eventId, isDraggable, timelineId }),
+    () =>
+      getSummaryRows({ browserFields, data, eventId, isDraggable, timelineId }).filter(
```

Comment:
could we check `if the value exists` in `getSummaryRows`, so we could avoid casting them in line 307, 308.
and since that `getSummaryRows` is getting bigger, probably we could move it to another file, too.